### ISSUE TITLE
A11y View links in Teacher Tips should be updated to 'latest' when a sim is republished

### ISSUE THREAD

**Issue by BLFiedler** [phetsims/a11y-research#173]
From Slack conversation:

Amy Rouinfar  10:42 AM
I saw the new Interactive Description video that went out in the email this morning. Really nicely done! I like how there’s now an Inclusive Features section of the Teacher Tips too. One thing I noticed is that the link to the sim’s a11y view is versioned, meaning it doesn’t use latest in the path like published sims. This means that the a11y view link will need to be manually maintained. It tried manually creating the URL to see if there is an a11y view under latest but it doesn’t seem like it. For example, https://phet.colorado.edu/sims/html/ratio-and-proportion/latest/ratio-and-proportion_a11y_view.html. I’m not sure if there were already conversations about this, but I can bring it up in website meeting.

Michael Kauzmann Today at 10:59 AM
We do not support a11y views on phet.colorado.edu. They must be used from phet-dev.

Jesse Greenberg  1 hour ago
Historically that is true, but hopefully they can be used on phet-server after https://github.com/phetsims/chipper/issues/1143 (nov 2021)

Michael Kauzmann  1 hour ago
First I'm hearing of it! We did something very similar for deploying PhET-iO simulations.
But I don't know what it's supposed to look like:
https://phet.colorado.edu/sims/html/ratio-and-proportion/latest/ratio-and-proportion_a11y_view.html

Michael Kauzmann  1 hour ago
Let me know if you want to discuss how we detected relative paths for PhET-iO deploys.

Jesse Greenberg  1 hour ago
OK thanks! Here is how I did it: https://github.com/phetsims/chipper/blob/a0fbf13c789a2fac2bf04f618b74a6889cab06fa/templates/sim-a11y-view.html#L226-L230
Was added in november, looks like RaP 1.0 was published in may so I wouldn't expect it to work there.

Michael Kauzmann  1 hour ago
Was there an MR for it?

Michael Kauzmann  1 hour ago
Looks like it only supports english? Why not the "all" html? Looks like it would only match on hostname of phet.colorado.edu. Do you know if that changes when you launch something in an iframe?
What sim would work on phet.colorado.edu right now?

Jesse Greenberg  1 hour ago
There was not, it was discussed but determined not to be worth the time.

Michael Kauzmann  1 hour ago
Gotcha! I miss read some commit messages, sorry

Jesse Greenberg  1 hour ago
 Looks like it only supports english? Why not the "all" html?
I am not too worried about this for a11y views at the moment.
Do you know if that changes when you launch something in an iframe?
That is a good question, I am not sure.

Jesse Greenberg  1 hour ago
What sim would work on phet.colorado.edu right now
Anything published after Nov 2021, not sure if there is one.

Michael Kauzmann  1 hour ago
Hmmm, so the video shows the a11y view for JT but JT doesn't work with the a11y view?

Amy Rouinfar  1 hour ago
Should we MR the sims with interactive description so that the a11y view links that we are publicly providing will be on phet-server?

Jesse Greenberg  1 hour ago
so the video shows the a11y view for JT but JT doesn't work with the a11y view?
Not under latest. For the purposes the course/Interactive Description video it was decided to just continue using phet-dev for existing links.

Jesse Greenberg  1 hour ago
Should we MR the sims with interactive description so that the
The a11y team decided it wasn't worth the time, but we certainly could if that needs to change!

Amy Rouinfar  43 minutes ago
Ok, good to know. Might want to make a note somewhere to manually update the a11y view link to latest when a sim is republished off of master.

**Comment by BLFiedler** [comment]
To be brought up at the next monthly check in or Q2 planning
